cmdstanpy/install_cmdstan.py

```diff
@@ -349,12 +349,13 @@ def compile_example(verbose: bool = False) -> None:
 
     :param verbose: Boolean value; when ``True``, show output from make command.
     """
-    cmd = [
-        MAKE,
-        Path(
-            os.path.join('examples', 'bernoulli', 'bernoulli' + EXTENSION)
-        ).as_posix(),
-    ]
+    path = Path(
+        os.path.join('examples', 'bernoulli', 'bernoulli' + EXTENSION)
+    )
```

Comment:
```suggestion
    path = (Path('examples') / 'bernoulli' / 'bernoulli').with_suffix(EXTENSION)
```
Rather than mixing os.path and pathlib